app/services/concerns/users/creneaux_wizard_concern.rb

```diff
@@ -64,6 +64,10 @@ def services
     @services ||= matching_motifs.includes(:service).map(&:service).uniq.sort_by(&:name)
   end
 
+  def follow_up_motifs?
+    @follow_up_motifs ||= Motif.where(service: services).exists?(follow_up: true, deleted_at: nil)
```

Comment:
👍 C'est intéressant d'utiliser `Motif.where(service: services)` plutôt que `matching_motifs`. On pourrait même restreindre un peu plus en ajoutant `where.not(bookable_by: :agents)` et en filtrant sur le territoire (plusieurs territoires peuvent utiliser le même service)

Reply:
Yes, j'avais hésité à pousser un peu + loin avec cette histoire de bookable_by et je me disais qu’en première étape, c'était déjà pas mal, mais vu ton retour, je vais peut-être pousser un peu plus :)